docs/releases/3.0.txt

```diff
@@ -287,6 +287,10 @@ Models
   customize the get and set behavior by overriding their
   :py:ref:`descriptors <descriptors>`.
 
+* Added :class:`~django.db.models.SmallAutoField` which acts much like an
+  :class:`~django.db.models.AutoField` except that it is guaranteed
```

Comment:
`that it is guaranteed ... ` this is true only on PostgreSQL and MySQL.

Reply:
Again, this was basically a copy of the release note when `BigAutoField` was introduced. Will change it to reflect that we are not providing the same guarantees.